### ISSUE TITLE
Concepts to communicate with distance/position sliders?

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab#107]
**On Focus:**
- m1 distance, slider. {{3 meter mark}}, {{somewhat close to}} m2, vectors {{small}}, forces {{0.041 712}} micronewtons.

**As m1 moves right decreasing distance**
- {{3.2 meter mark}}. Right, closer to m2, vectors bigger, forces now {{force value}} micronewtons.
- {{3.4 meter mark}}. Closer to m2, vectors get bigger, forces now {{force value}} micronewtons}}.

**As m1 moves left increasing distance**
- {{3.2 meter mark}}. Left, farther from m2, vectors smaller, forces now {{force value}} micronewtons.
- {{3 meter mark}}. Farther from m2, vectors smaller, forces now {{force value}} micronewtons.

@emily-phet, what do you think?

**Comment by terracoda** [comment]
Oops, hit enter prematurely, updating previous comment.

**Comment by terracoda** [comment]
@emily-phet or @arouinfar what do you think of the above position alerts?
It's a similar approach as in #106, but with direction added in. There should be curly braces around the direction string as they are dynamic, uttered only when there is a change in direction:

-  {{3.2 meter mark}}. {{Right,}} closer to m2, vectors bigger, forces now {{force value}} micronewtons
- continuing right: {{3.4 meter mark}}. Closer to m2, vectors get bigger, forces now {{force value}} micronewtons}}.

**Comment by terracoda** [comment]
Actually, the directional descriptions are a somewhat different approach from the Mass alerts which are more relational. My first thought for position changes was also more relational. For example:

**m1 moves closer to m2:**
- m1 position, slider. {{3.2 meter mark}}. As m1 gets {{closer to}} m2, vectors get bigger, forces now {{0.041 712 micronewtons}}.

**m1 moves further from m2**
- m1 position, slider. {{2.8 meter mark}}. As m1 gets {{further from}} m2, vectors get bigger, forces now {{0.041 712 micronewtons}}.

But then I re-arranged the alert to include direction (only on directional changes), and I am able to make the alert shorter. For example:

**m1 moves closer to m2 (2 default steps)**
- {{3.2 meter mark}}. {{{{Right}}, closer to m2}}, vectors bigger, forces now {{force value}} micronewtons.
- {{3.4 meter mark}}. {{Closer to m2}}, vectors get bigger, forces now {{force value}} micronewtons}}.

**m1 moves further from m2 (2 default steps)**
- {{3.2 meter mark}}. {{{{Left}}, farther from m2}}, vectors smaller, forces now {{force value}} micronewtons.
- {{3.0 meter mark}}. {{Farther from m2}}, vectors smaller, forces now {{force value}} micronewtons.

**Design Question:**
Do you have a preference for "relational" or "directional" position alerts?

**Comment by emily-phet** [comment]
@terracoda I really like the second set above - the 'directional' style alerts. They're definitely shorter, and I think provide relevant information without sound like they're trying to 'tell' you the patter you're supposed to be inferring. I'm imaging if you hit the arrow key moderately quickly, repeatedly (with some alerts interrupting each other), you'd hear something like 'vectors smaller - vectors smaller - vectors smaller ' or 'vectors larger, vectors larger, vectors larger' which feels like what we want to go for as opposed to 'as m1 gets further - as m1 gets further - as m1 gests further vectors get bigger...'.

**Comment by terracoda** [comment]
I'll update the design document and then close this issue. Thanks @emily-phet.

**Comment by terracoda** [comment]
The design document has been updated. Example alerts for initial values for position and changing position are in section [Changing the Position of a Sphere](https://docs.google.com/document/d/1-37qAgde2XrlXBQae2SgjartM35_EnzDD9pdtd3nXAM/edit#heading=h.d6mb64l644i)

I'm closing this issue. These alerts are ready for implementation.
